jestBabelTransform.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable import/no-extraneous-dependencies */
+
```

Comment:
can we remove this line?

Reply:
f5b221369bc9aa656c6e9d673b630f4049ed618f